src/app/cli/src/cli.rs

```diff
@@ -367,6 +368,27 @@ pub struct Delete {
     #[arg(value_parser = parsers::dataset_ref_pattern)]
     pub dataset: Vec<odf::DatasetRefPattern>,
 }
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/// Export a dataset
+#[derive(Debug, clap::Args)]
+#[command(after_help = r#"
+This command exports a dataset to a file(s) of a given format.
+TODO: add extend doc
+"#)]
+pub struct Export {
+    #[arg(index = 1)]
+    pub dataset: String,
+
+    #[arg(long)]
+    pub output_path: String, //todo pathbuf
+
+    #[arg(long, value_parser = ["parquet", "json", "csv"])]
+    pub output_format: String, //todo value_parser = ["csv", ...]
+
+    #[arg(long)]
+    pub partition_size: Option<usize>,
```

Comment:
Unclear if it's in bytes or records

Reply:
I plan to add params documentation later in this PR

Reply:
It's not about the docs, it's about the name

Reply:
but name was take from ticket description: https://github.com/kamu-data/kamu-cli/issues/353
should we change it to something like `--rows-per-file`?

Reply:
Yes, the switch is unclear similarly. Maybe like this or maybe `--records-per-file`.